pgpq.go

```diff
@@ -153,3 +154,7 @@ func coalesceTime(t1, t2 time.Time) time.Time {
 	}
 	return t2
 }
+
+func unsafeString(p []byte) string {
+	return unsafe.String(unsafe.SliceData(p), len(p))
```

Comment:
this requires Go 1.20+

Reply:
Couldn't it be just `[]byte(theJsonRawMessageVar)`?

Reply:
nope. tried, doesn't work, strings only :shrug: